ggshield/verticals/secret/output/schemas.py

```diff
@@ -14,6 +14,8 @@ class FlattenedPolicyBreak(BaseSchema):
     incident_url = fields.String(required=True, dump_default="")
     incident_details = fields.Nested(SecretIncidentSchema)
     known_secret = fields.Bool(required=True, dump_default=False)
+    is_ignored = fields.Bool(required=False)
+    ignore_reason = fields.String(required=False)
```

Comment:
I would prefer if we used a separate schema for IgnoreReason. If we do so we don't need the `is_ignored` field: either the `ignore_reason` field is there and the secret has been ignored, or it's not and the secret has not been ignored.

Reply:
Made it a separate schema